bigquery/samples/tests/test_client_query_dry_run.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .. import client_query_dry_run
+
+
+def test_client_query_dry_run(capsys, client):
+
+    query_job = client_query_dry_run.client_query_dry_run(client)
+    out, err = capsys.readouterr()
+    assert "This query will process 65935918 bytes." in out
```

Comment:
That's a bit too specific. Can we check with a regex that it's non-zero instead?